flink-yarn/src/test/java/org/apache/flink/yarn/FlinkYarnSessionCliTest.java

```diff
@@ -124,27 +124,6 @@ public void testCorrectSettingOfMaxSlots() throws Exception {
 		assertEquals(1, clusterSpecification.getNumberTaskManagers());
 	}
 
-	@Test
-	public void testCorrectSettingOfDetachedMode() throws Exception {
```

Comment:
I would leave this test and simply do:

```
		assertTrue(!executorConfig.get(DeploymentOptions.ATTACHED));
```

at the end, without going through the descriptor. Simply to verify that the `-yd` flag is correctly propagated.

Reply:
Thanks for your insight! Integrated and will merge once travis gives green.